README.md

```diff
@@ -72,8 +73,6 @@ NOTE: This may require a computer restart to take effect.
   1. Add a record to your local `hosts` file for `127.0.0.1 cep.test`
       - `sudo vim /etc/hosts`
 
-     _WARNING: This name **must** match the **agave callback URL** defined for the client in `settings_secret.py` for `_AGAVE_TENANT_ID`._
```

Comment:
I do **not** see `AGAVE_TENANT_ID` in `settings_secret.example.py`.

I **do** see `AGAVE_TENANT_ID` in Stache secrets, but it is `portals` not `cep.dev`.

Reply:
I'll fix this